src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.Service/Properties/Resources.resx

```diff
@@ -139,7 +139,10 @@
     <value>Restart (Ctrl+R)</value>
   </data>
   <data name="wox_plugin_service_restarted_notification" xml:space="preserve">
-    <value>{0} has been restarted</value>
+    <value>The service has been restarted</value>
+  </data>
+  <data name="wox_plugin_service_restart_error_notification" xml:space="preserve">
```

Comment:
I guess I missed the `wox` prefix in the PR that introduced this plugin.
For future reference, don't use `wox` as a prefix.

Reply:
Btw, even the entire prefix `wox_plugin_` is pointless since this resource file is for this plugin only.
We should come up with a proper convention for naming the resource strings Id(s).

Reply:
I agree. The problem is not related only to this plugin.
Don't know what's the impact of changing resource names for localization.

Reply:
I think we should open a general issue. Because this problem exists for all plugins,  PT Run itself and the Plugin Manager branch too.

Reply:
@davidegiacometti 
correct, localization is impacted and we need to coordinate before changing existing id(s).